buildSrc/src/main/kotlin/pklJavaLibrary.gradle.kts

```diff
@@ -56,7 +82,11 @@ tasks.compileKotlin { enabled = false }
 
 tasks.jar {
   manifest {
-    attributes += mapOf("Automatic-Module-Name" to "org.${project.name.replace("-", ".")}")
+    attributes +=
+      mapOf(
+        "Automatic-Module-Name" to "org.${project.name.replace("-", ".")}",
+        "Add-Exports" to jpmsExports.joinToString(" ") { it.substringBefore("=") },
+      )
```

Comment:
`Add-Exports` within a JAR manifest has a weird syntax. It's _just_ the `<module>/<package>` portion of the `--add-exports=*` argument, before the second equals sign, and `ALL-UNNAMED` is implied as a value. Multiple modules are space-separated. Here's what the final thing looks like:

![image](https://github.com/user-attachments/assets/e37e12e0-da78-4fdd-997a-134f89c17ced)

After adding this attribute:
```
> Task :pkl-cli:testStartJavaExecutableJdk17
Pkl 0.28.0-dev+9e27cbd (Linux 5.15.167.4-microsoft-standard-WSL2, Java 17.0.9)
```